Tests/ClientTests/Ecosia/WhatsNewLocalDataProviderTests.swift

```diff
@@ -43,18 +43,6 @@ final class WhatsNewLocalDataProviderTests: XCTestCase {
     
     // MARK: - Upgrade Tests
     
-    func testUpgradeToDifferentVersionShouldShowWhatsNew() {
```

Comment:
I don't get why this is no longer relevant, can you explain it?

Reply:
From what I've seen then you were able to keep it and make it reliable by injecting the user then, is that correct?

Reply:
Correct! Applied the change to the `EcosiaInstallType` as well